src/content/contributing/adding-staking-product/index.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Adding staking products or services
```

Comment:
Overall I think this listing policy is solid! One question in my mind is: what is must-have vs. what is nice-to-have? I don't think there's a ton of clarity on what aspects are deal-breakers in terms of getting listed or not. Perhaps we don't need to establish a hard line _yet_ given how nascent the space is but I think that would be ideal for clarity around the decision criteria.

Reply:
I think we can at least clarify some of these, e.g. in my mind

Must-have:
- open source 
- security audit

Nice-to-have:
- client diversity metrics
- ability to choose clients
- bug bounty program

Not sure:
- out of beta

Reply:
@samajammin I'm hesitant to frame it this way since currently we would not be following those guidelines. Many products are not yet formally audited. The new design is intended to allow flexibility and transparency here so users can decide for themselves.

I think for v1 its best we phrase this as a multifactorial decision that is ultimately at our discretion, but decisions are based on x, y, z (ie, open source, audits, battle testing, etc) and the more boxes you check the better. Have some changes I'll push in a moment